pkgs/flox/default.nix

```diff
@@ -97,15 +99,24 @@ in
           --bash <($out/bin/flox --bpaf-complete-style-bash) \
           --fish <($out/bin/flox --bpaf-complete-style-fish) \
           --zsh <($out/bin/flox --bpaf-complete-style-zsh)
+
+        wrapProgram "$out/bin/flox"                                       \
+                    --prefix PATH :                                       \
+                      "${flox-bash}/bin:${flox-bash}/libexec:${git}/bin"
       '';
 
       doInstallCheck = true;
       postInstallCheck = ''
         # Quick unit test to ensure that we are not using any "naked"
         # commands within our scripts. Doesn't hit all codepaths but
         # catches most of them.
-        git init;
-        env -i USER=`id -un` HOME=$PWD $out/bin/flox --debug envs > /dev/null
+
+        set -eu;
+        set -o pipefail;
+
+        env -i USER=`id -un` "HOME=$PWD" "$out/bin/flox" --help > /dev/null||  \
+          env -i USER=`id -un` "HOME=$PWD" "$out/bin/flox" --debug --verbose   \
+                                                           --help
```

Comment:
Looks like leftover debugging?

Also why are we just running help? IIRC the choice of the `envs` subcommand was intentional, to exercise more codepaths and in particular exercise the hashing of all required commands.